omniauth-openid.gemspec

```
@@ -2,7 +2,7 @@
 require File.expand_path('../lib/omniauth-openid/version', __FILE__)
 
 Gem::Specification.new do |gem|
-  gem.add_dependency 'omniauth', '~> 1.0'
+  gem.add_dependency 'omniauth', ['>= 1.0', '< 3.0']
```

Let's just go with a pessimistic 2.0 here just in case
```suggestion
  gem.add_dependency 'omniauth', '~> 2.0'
```

---

Actually on second thought let's leave it permissive since there are no real breaking changes here.